tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -5237,6 +5238,7 @@ def globalReadGuardK(self, kernel, tP):
 
     def globalReadGuardKBody(tP):
       tc = tP["tensorChar"]
+      tc2 = "" if tc == "Metadata" else tc
```

Comment:
can we deprecate tc2, and just replase tc as "" for Metadata

Reply:
Chaaged to tcDataType instead